### PR TITLE
added template for next minor

### PR DESCRIPTION
**Pull request by keulinho**

### CODE REVIEW

File: src/6.6/6.6.2.0.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+### Added meteor component library to the administration
+
+The [Meteor component library](https://shopware.design/meteor-components/) was added as a new dependecy to the administration.
+All components from the library are now also directly available in the administration.
+
+### Sync option for CLI theme commands
+
+The `theme:compile` and `theme:change ` command now accept `--sync` option to compile themes synchronously. The `--sync` option is useful for CI/CD pipelines, when at runtime themes should be compiled async, but during the build process you want sync generation.
+
+### Order Approval improvements
```
Comment (keulinho) [inline src/6.6/6.6.2.0.md:32]: @kermie this feature is part of the commercial plugin in the b2b context, should we add a note or something about that?

Comment (kermie) [reply src/6.6/6.6.2.0.md:44]: Yep. Would do like this:

### Order Approval improvements (valid for commercial plugin in B2B context)

**Review by bojanrajh — APPROVED** [review]

**Comment by bojanrajh** [comment]
Closing because this repo requires signed commits (since when?) @shyim 
https://github.com/shopware/release-notes/pull/14

**Comment by keulinho** [comment]
> Closing because this repo requires signed commits (since when?) @shyim #14

Not sure there is no setting in GH that i can find :thinking: 

But you can force merge even when red, that's what i did with the MR from today

**Pull request closed without merging**